conduit/ChangeLog.md

```diff
@@ -1,3 +1,7 @@
+## 1.2.6.7
```

Comment:
This would be a minor version bump, not a patch, since it's not forwards-compatible (i.e., 1.2.7).